### PR TITLE
travis: test on 386

### LINKED ISSUE #1444: gonum fails to build when ints are 32 bits
Running tests with GOARCH=386 results in:

```
# gonum.org/v1/gonum/graph/encoding/digraph6 [gonum.org/v1/gonum/graph/encoding/digraph6.test]
./digraph6.go:65:9: constant 68719476736 overflows int
FAIL	gonum.org/v1/gonum/graph/encoding/digraph6 [build failed]
```

And same problem in graph6.

### What version of Go and Gonum are you using?

Tested with 0.8.1 and HEAD (1a536f9a771e1c098630abc9a8bf53b7e5e1994f) with go1.15 on linux

### Does this issue reproduce with the current master?

Yes

This simple patch fixes it but doesn't seem very elegant

```
diff --git a/graph/encoding/digraph6/digraph6.go b/graph/encoding/digraph6/digraph6.go
index 5b53136f..f172e41e 100644
--- a/graph/encoding/digraph6/digraph6.go
+++ b/graph/encoding/digraph6/digraph6.go
@@ -62,7 +62,7 @@ func Encode(g graph.Graph) Graph {
 		buf.WriteByte(byte(n) + 63)
 	case n < 258048:
 		buf.Write([]byte{126, byte(n>>12) + 63, byte(n>>6) + 63, byte(n) + 63})
-	case n < 68719476736:
+	case int64(n) < 68719476736:
 		buf.Write([]byte{126, 126, byte(n>>30) + 63, byte(n>>24) + 63, byte(n>>18) + 63, byte(n>>12) + 63, byte(n>>6) + 63, byte(n) + 63})
 	default:
 		panic("digraph6: too large")
diff --git a/graph/encoding/graph6/graph6.go b/graph/encoding/graph6/graph6.go
index 90c84117..18153a83 100644
--- a/graph/encoding/graph6/graph6.go
+++ b/graph/encoding/graph6/graph6.go
@@ -63,7 +63,7 @@ func Encode(g graph.Graph) Graph {
 		buf.WriteByte(byte(n) + 63)
 	case n < 258048:
 		buf.Write([]byte{126, byte(n>>12) + 63, byte(n>>6) + 63, byte(n) + 63})
-	case n < 68719476736:
+	case int64(n) < 68719476736:
 		buf.Write([]byte{126, 126, byte(n>>30) + 63, byte(n>>24) + 63, byte(n>>18) + 63, byte(n>>12) + 63, byte(n>>6) + 63, byte(n) + 63})
 	default:
 		panic("graph6: too large")
```

### PR DESCRIPTION
**Pull request by kortschak**
Depends on #1505 and #1506.

Please take a look.

Closes #1444.

### CODE REVIEW

**Review by sbinet — APPROVED** [review]
LGTM

**Comment by kortschak** [comment]
This will need to be rebased after the dependency PRs are merged.

**Comment by kortschak** [comment]
Turns out that 386 is not an arch on travis. Will need to use `GOARCH=386` in an env.

**Comment by kortschak** [comment]
... and travis doesn't respect setting `GOARCH`.

File: .travis/script.d/test.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Travis clobbers GOARCH env vars so this is necessary here.
+if [[ -n $FORCEGOARCH ]]; then
```
Comment (sbinet) [inline .travis/script.d/test.sh:4]: shouldn't this read:
```suggestion
if [[ -n $FORCE_GOARCH ]]; then
```
instead ?

**Review by sbinet — APPROVED** [review]

File: .travis/script.d/test.sh (hunk 1 above)
Comment (kortschak) [reply .travis/script.d/test.sh:4]: Indeed it should. I thought I got all of them.

**Comment by kortschak** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+# Gonum [![Build Status](https://www.travis-ci.com/gonum/gonum.svg?branch=master)](https://www.travis-ci.com/gonum/gonum/branches) [![Build status](https://ci.appveyor.com/api/projects/status/valslkp8sr50eepn/branch/master?svg=true)](https://ci.appveyor.com/project/Gonum/gonum/branch/master) [![codecov.io](https://codecov.io/gh/gonum/gonum/branch/master/graph/badge.svg)](https://codecov.io/gh/gonum/gonum) [![coveralls.io](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum) [![Go Report Card](https://goreportcard.com/badge/github.com/gonum/gonum)](https://goreportcard.com/report/github.com/gonum/gonum) [![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
```
Comment (sbinet) [inline README.md:1]: perhaps for another PR, but we could perhaps split this very looooooong line of badges into a set of lines, each badge its own.
if there aren't blank lines in between the visual rendering would be the same (but limit git-diff-noise)

**Review by sbinet — APPROVED** [review]
LGTM (modulo green travis)

File: README.md (hunk 2 above)
Comment (kortschak) [reply README.md:1]: That sounds like a good idea.

**Pull request merged**